### PR TITLE
Add footer and page Sobre nosotros

### LINKED ISSUE #8: Crear footer y nosotros
### Explanation
Crear parte de "nosotros"

### Screenshots
image

### PR DESCRIPTION
**Pull request by Jesus-Gonzalez-Arroyo**
fixes #8 

Agregando el footer y parte de nosotros al poryecto

### CODE REVIEW

**Comment by elias-miguel-vergara** [comment]
espero aprobacion de @Israel-Laguan

File: tailwind.config.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
+    "./pages/*/.{js,ts,jsx,tsx}",
```
Comment (Israel-Laguan) [inline tailwind.config.js]: Please revert this change

File: pages/Nosotros.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Head from "next/head";
+import Footer from "../Components/Footer";
+import SobreNosotros from "../Components/SobreNosotros"
+
+function Nosotros() {
```
Comment (Israel-Laguan) [inline pages/Nosotros.jsx]: Change to an arrow function

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]
Revert the glob changes

**Comment by Israel-Laguan** [comment]
Build failed, please check

File: styles/SobreNosotros.module.css [hunk 3]
```diff
@@ -1,23 +1,27 @@
 .container_sobrenosotros {
-    @apply w-4/5 m-auto flex items-center my-4; 
+    @apply w-full m-auto; 
 }
 
 .container_text {
-    @apply w-1/2;
+    @apply w-11/12 m-auto mb-12 max-w-5xl rounded-2xl bg-[#125C13] flex flex-wrap items-center sm:w-3/5 sm:flex-nowrap sm:bg-transparent sm:w-4/5;
```
Comment (Israel-Laguan) [inline styles/SobreNosotros.module.css]: Please add `#125C13` to color variables in tailwind

File: styles/SobreNosotros.module.css [hunk 4]
```diff
@@ -1,23 +1,27 @@
...
 }
 
 .container_child {
-    @apply w-4/5 bg-[#125C13] m-auto text-center p-5 text-[#fff] leading-8 rounded-xl;
+    @apply w-full bg-[#125C13] m-auto text-center p-8 text-[#fff] leading-8 rounded-xl;
```
Comment (Israel-Laguan) [inline styles/SobreNosotros.module.css]: I think `#fff` is already available as just `white`

Reference: https://tailwindcss.com/docs/customizing-colors

File: components/SobreNosotros.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+const SobreNosotros = () => (
+    <div className={Style.container_sobrenosotros}>
+        <div className={Style.container_text}>
+            <div className={Style.container_child}>
+                <p className={Style.title_text}>Sobre Nosotros</p>
+                <p className={Style.text}>Somos un grupo de personas enfocadas a ofrecer orientacion y guia a los Juniors, proporcionando oportunidades de aprendizaje con nuestros sistema de proyectos, facilitando la comunicacion y brindando el apoyo necesario para que las experiencias sean lo mas satisfactorias y gratificantes, como tambien fomentar la confianza de sus conocimientos, para poder avanzar profesionalmente.</p>
+            </div>
+            <div className={Style.container_Image}>
+                <Image
```
Comment (joselyepez) [inline components/SobreNosotros.jsx:14]: add alt attribute

File: pages/_app.js [hunk 6]
```diff
@@ -1,12 +1,16 @@
 import Navbar from "../components/Navbar";
 import Main from "../components/Main";
+import Footer from "../Components/Footer";
+import SobreNosotros from "../Components/SobreNosotros"
```
Comment (joselyepez) [inline pages/_app.js]: add ;

File: pages/index.js [hunk 7]
```diff
@@ -1,6 +1,8 @@
 import Head from "next/head";
 import Navbar from "../components/Navbar";
 import Main from "../components/Main";
+import Footer from "../Components/Footer";
+import SobreNosotros from "../Components/SobreNosotros"
```
Comment (joselyepez) [inline pages/index.js]: add ;

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
It's not building, please check
Screenshot_20230119_170842_Chrome

**Review by elias-miguel-vergara — APPROVED** [review]

**Review by joselyepez — APPROVED** [review]

**Comment by Israel-Laguan** [comment]
image

image

For reference this is how it looks on my side

**Comment by Jesus-Gonzalez-Arroyo** [comment]
In the responsive part, we have to wait for Jose Luis to fix a part of the index since it generates a conflict in the responsiveness of the other components

File: styles/SobreNosotros.module.css [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+.container_child {
+    @apply w-full bg-[darkgreen] m-auto text-center p-8 text-[white] leading-8 rounded-xl;
+}
+
+.text {
+    @apply w-full font-light;
+}
+
+.title_text {
+    @apply text-2xl text-center text-[#fff] my-3 font-extrabold;
```
Comment (Israel-Laguan) [inline styles/SobreNosotros.module.css]: Use white for this color

**Comment by Jesus-Gonzalez-Arroyo** [comment]
Conflictos resueltos

**Review by Israel-Laguan — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**